src/core/sagas/search.js

```diff
@@ -18,7 +19,8 @@ export function* fetchSearchResults({ payload }) {
   yield put(errorHandler.createClearingAction());
 
   try {
-    const { filters, page } = payload;
+    const { filters } = payload;
+    const page = parsePage(filters.page);
```

Comment:
I think this would be better set in the reducer though, actually. I'll make sure there's test coverage for it but I think it should be there rather than in the saga.